### ISSUE TITLE
Not able to open app on OS X

### ISSUE THREAD

**Issue by pablopunk** [Aluxian/electron-superkit#7]
Hi I'm not really sure if I've done everything correctly, but when I run `gulp build:darwin64` I get this error:
`'download:darwin64' errored after 2.09 s
[06:39:17] Error: ENOENT, rename './build/darwin64/Electron.app'
  at Error (native)`

**Update**: I downloaded the repo again, copied the files and now I can build it. But I have another question: Is there any way I can change where the files are placed? I mean, when I build the app it has html, scripts, and styles folders by default, but I want to have all my javascript and css files in the html folder (as it's where my app looks for them).

After a simple build I can just `cp -R` all files into that folder, but this is not possible when I package the app (as there's no `resources/app` folder anymore) 

**Update 2**: Ok, I answer to my question again, I managed to change the gulp tasks in order to build as I said, but now I have another problem (such a noob, I know):

I package the app with `gulp pack:darwin64` but when I start the app from /Applicactions it doesn't open. If I open it with the `open` command it shows `LSOpenURLsWithRole() failed with error -10810 for the file /Applications/Chronocube.app`.

The weird thing is that if I run the binary `/Applications/Chronocube.app/Contents/MacOS/Chronocube`everything works, but I'm still not able to run by opening the app normaly.

**Comment by aluxian** [comment]
1. For changing paths that's the solution, to change the gulp tasks. One of the goals of my kit is to help automate everything.
2. Does the app work if you move it somewhere else than `/Applications/`?
3. Try (this will set permissions to make it executable):
   
   ```
   chmod +x /Applications/Chronocube.app
   chmod +x /Applications/Chronocube.app/Contents/MacOS/Chronocube
   ```
4. Does it work on another OS X machine?
5. Google for `LSOpenURLsWithRole() failed with error 10810 for the file`, do the first results help? Especially the first 4, as I've just looked at them myself.

**Comment by pablopunk** [comment]
1. Yep, great job ;)
2. I did literally nothing and **now it worked**. Maybe a reboot was required, don't know why, but now it runs.
3. I'm answering this anyway, just in case people search for this question in the future. And no, the first thing I did when I watched the error was `chmod` and it didn't work.
4. Didn't try but if now works, it should do it on any mac I guess.
5. Most of those results were because of a JVM problem, so they didn't help me.

As it's already **solved**, we can close the issue, I just wanted to keep this info here. Great repo :+1:
